docs/content/preview/architecture/docdb-replication/async-replication.md

```diff
@@ -14,141 +14,580 @@ menu:
+
+In that case, we need to bootstrap the replication process by first
+copying the source universe to the target universe.
+
+Today, this is done by backing up the source universe and restoring it
+to the target universe.  In addition to copying all the data, this copies
+the table schemas so they are identical on both sides.  Before the
+backup is done, the current Raft log IDs are saved so the replication
+can be started after the restore at a point before the backup was done.
+This ensures any data written to the source universe during the backup
+is replicated.
+
+Ongoing work,
+[#17862](https://github.com/yugabyte/yugabyte-db/issues/17862), will
+replace using backup and restore here with directly copying RocksDB
+files between the source and target universes.  This will be more
+performant and flexible and remove the need for external storage like S3
+to set up replication.
 
-For details about configuring an xCluster deployment, see [xCluster deployment](../../../deploy/multi-dc/async-replication).
-
-xCluster replication of data works across YSQL and YCQL APIs because the replication is done at the DocDB level.
 
 ## Supported deployment scenarios
 
-A number of deployment scenarios is supported.
+xCluster currently supports active-passive and active-active
+deployments.
 
 ### Active-passive
 
-The replication could be unidirectional from a source universe (also known as producer universe) to one target universe (also known as consumer universe or sink universe). The target universes are typically located in data centers or regions that are different from the source universe. They are passive because they do not take writes from the higher layer services. Usually, such deployments are used for serving low-latency reads from the target universes, as well as for disaster recovery purposes.
+Here the replication is unidirectional from a source universe to a target
+universe.  The target universe is typically located in data centers or
+regions that are different from the source universe. The target universe
+is passive because it does not take writes from the higher layer
+services.
+
+Usually, such deployments are used for serving low-latency reads from
+the target universes, as well as for disaster recovery purposes.  When
+used primarily for disaster recovery purposes, these deployments are
+also called active-standby because the target universe stands by to take
+over if the source universe is lost.
+
+Either transactional or non-transactional mode can be used here, but
+transactional mode is usually preferred because it provides consistency
+if the source universe is lost.
 
-The following diagram shows the source-target deployment architecture:
+The following diagram shows an example active-passive deployment:
 
-<img src="/images/architecture/replication/2DC-source-sink-deployment.png" style="max-width:750px;"/>
+<img src="/images/architecture/replication/active-standby-deployment-new.png" style="max-width:750px;"/>
```

Comment:
Could we change this to markdown style  image embed as 

`[alt title](image path)`  ..

We need to do this as the image is not resizing properly when the page is resized.

Reply:
Same for the other image